backend/src/plugins/authorization/authorization.plugin.ts

```diff
@@ -29,7 +30,7 @@ const auth: FastifyPluginAsync<Options> = async (fastify, opts) => {
         (route) => route === request.routerPath,
       );
 
-      if (isWhiteRoute) {
+      if (isWhiteRoute && request.method === HttpMethod.GET) {
```

Comment:
What about `SIGN_IN` and `SIGN_UP`?

Reply:
Good point, they won't work with this solution
Initially, I wanted to add a method for each white route and check the method in authorization plugin, but Andrey suggested to check for GET requests